### PR TITLE
Story/jira 92 ImageCompression 

### PR DESCRIPTION
**Pull request by rajguptaH**

### CODE REVIEW

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/TestingAndCalibrationService/GoogleDriveService.cs [hunk 1]
```diff
@@ -91,7 +94,7 @@ private DriveService GetService()
             UserCredential credential;
 
             //TODO: need to see if the client secret file can be stored in some other place.
-            using (var stream = new FileStream(Path.Combine(_hostingEnvironment.WebRootPath, "client_secret_612092452145-21d21u1m196soc5t92j3vagr8rf7h8u7.apps.googleusercontent.com.json"),
+            using (var stream = new FileStream(Path.Combine(_hostingEnvironment.WebRootPath, "client_secret_373443832187-a5fj833jc592dm3unjuan02ekoecv67t.apps.googleusercontent.com.json"),
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/TestingAndCalibrationService/GoogleDriveService.cs:97]: 1. client secret file should not be hard coded in code.
2. it should not be placed under wwwroot.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/TestingAndCalibrationService/FileCompressionService.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        {
+            string ImageUrl;
+
+
+            ImageUrl = file.FileName;
+            string FilePathDelete = Path.Combine(_WebHostingEnviroment.WebRootPath, _configuration["DownloadData:FolderName"], file.FileName);
+            if (File.Exists(FilePathDelete))
+            {
+                File.Delete(FilePathDelete);
+            }
```
Comment (smartdusttechnologies) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/TestingAndCalibrationService/FileCompressionService.cs:56]: why do we need this logic?

Comment (rajguptaH) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/TestingAndCalibrationService/FileCompressionService.cs]: i have removed this logic

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/TestingAndCalibrationService/GoogleDriveService.cs (hunk 1 above)
Comment (rajguptaH) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Business/Services/TestingAndCalibrationService/GoogleDriveService.cs]: i have fixed all the things youhve telled me to do .

**Pull request merged**